app/web/components/AppRoute.tsx

```diff
@@ -130,10 +134,10 @@ export default function AppRoute({
             {/* Have to wrap this in a fragment because of https://github.com/mui-org/material-ui/issues/21711 */}
             <>{children}</>
           </Container>
-          {!noFooter && !isNativeEmbed && <Footer />}
+          {!noFooter && !isNativeEmbed && !isMobile && <Footer />}
         </>
       )}
-      {!isPrivate && !isNativeEmbed && <CookieBanner />}
+      {!isPrivate && !isNativeEmbed && !isMobile && <CookieBanner />}
```

Comment:
Minor thing again - but I wonder if it's worth extracting these to readable variable names? Maybe it's at the point where it's fine to leave for now unless we add one more `&&`

Reply:
Yeah that makes sense!